src/main/java/it/gov/pagopa/pu/workflow/service/WorkflowService.java

```diff
@@ -3,11 +3,15 @@
 import io.temporal.client.WorkflowStub;
 import it.gov.pagopa.pu.workflow.dto.generated.WorkflowStatusDTO;
 
+import java.time.Duration;
+
 public interface WorkflowService {
 
   <T> T buildWorkflowStub(Class<T> workflowClass, String taskQueue, String workflowId);
 
   WorkflowStub buildUntypedWorkflowStub( String taskQueue, String workflowId);
 
   WorkflowStatusDTO getWorkflowStatus(String workflowId);
+
+  <T> T buildWorkflowStubScheduled(Class<T> workflowClass, String taskQueue, String workflowId, Duration startDelay);
```

Comment:
let's add a version which accept a LocalDateTime, which will calculate the Duration and invoke the other method.
I think we will have more useCases where we will know the future date, this way we will centralize the calculation of the Duration
```suggestion
  <T> T buildWorkflowStubDelayed(Class<T> workflowClass, String taskQueue, String workflowId, Duration startDelay);
  <T> T buildWorkflowStubScheduled(Class<T> workflowClass, String taskQueue, String workflowId, LocalDateTime dateTime);
```

Reply:
ps: let's add a third method to accept an overloaded method which will accept an OffsetDateTime.
be careful to calculate it's Duration!

It's ok to calculate directly it:
`Duration.between(offsetDateTime, OffsetDateTime.now())`

But if you think to re-use the LocalDateTime method, you should keep attention on how you will convert it (you have to convert the offset to the system before to call the toLocalDateTime):
```
OffsetDateTime offsetDateTime = OffsetDateTime.now(ZoneOffset.UTC);
offsetDateTime.atZoneSameInstant(ZoneOffset.systemDefault()).toLocalDateTime();
```

let's test the conversion into Duration and the right call of the Duration based method (use MockitoSpyBean):
* in the LocalDateTime version let's give LocalDateTime.now().plusDays(1), let's expect that the arg of Duration based method could not be more than 4 secs:
```
Duration expectedMaxDuration = Duration.ofDays(1);
LocalDateTime nextSchedule = LocalDateTime.now().plus(expectedMaxDuration);

Duration argDuration = Duration.ofDays(1).minusSeconds(4);
Duration diff = expectedMaxDuration.minus(argDuration);
diff.toSeconds() > 0;
diff.toSeconds() <5; // we are expecting that the test could not execute for more than 4 seconds
```
* in the OffsetDateTime version let's use a different offset:
```
Duration expectedMaxDuration = Duration.ofDays(1);
OffsetDateTime nextSchedule = OffsetDateTime.now(ZoneOffset.MAX).plus(expectedMaxDuration);


Duration argDuration = Duration.ofDays(1).minusSeconds(4);
Duration diff = expectedMaxDuration.minus(argDuration);
diff.toSeconds() > 0;
diff.toSeconds() <5; // we are expecting that the test could not execute for more than 4 seconds
```